tools/crm_ticket.c

```diff
@@ -351,6 +351,43 @@ find_ticket_constraints(cib_t * the_cib, gchar *ticket_id, xmlNode ** ticket_con
     return rc;
 }
 
+PCMK__OUTPUT_ARGS("ticket-attribute", "gchar *", "const char *", "const char *")
+static int
+ticket_attribute_default(pcmk__output_t *out, va_list args)
+{
+    gchar *ticket_id G_GNUC_UNUSED = va_arg(args, gchar *);
+    const char *name G_GNUC_UNUSED = va_arg(args, const char *);
+    const char *value = va_arg(args, const char *);
+
+    pcmk__formatted_printf(out, "%s\n", value);
```

Comment:
This is only valid for text format, need to separate from default, or maybe just use info()

Reply:
Changed to use info().